src/main/tools/linux-sandbox-pid1.cc

```diff
@@ -460,11 +649,22 @@ int Pid1Main(void *sync_pipe_param) {
   if (opt.fake_hostname) {
     SetupUtsNamespace();
   }
-  MountFilesystems();
-  MakeFilesystemMostlyReadOnly();
-  MountProc();
+
+  if (opt.hermetic) {
+    MountSandboxAndGoThere();
+    CreateEmptyFile();
+    MountDev();
+    MountProc();
```

Comment:
This mounts /proc on /proc not on sandbox-root/proc; after SYS_pivot_root syscall it will be lost.